### ISSUE TITLE
Error after training model  

### ISSUE THREAD

**Issue by tpackolus** [facebookresearch/fairseq#5013]
## ❓ Questions and Help
After training model a get error 

```
python3.10/multiprocessing/resource_tracker.py:224: UserWarning: resource_tracker: There appear to be 8 leaked semaphore objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
```

I must kill process 

can help me ?

### Before asking:
1. search the issues. yes

3. search the docs. yes

#### What's your environment?

 - fairseq Version (e.g., 1.0 or main):  0.12.2
 - PyTorch Version (e.g., 1.0)  1.13.1+cu117
 - OS (e.g., Linux): Ubuntu 20
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version: 3.10
 - CUDA/cuDNN version: 11.4
 - GPU models and configuration:   --optimizer nag --lr 0.25 --clip-norm 0.1 --dropout 0.2 --max-tokens 4000 --arch fconv_iwslt_de_en --max-epoch 2 --save-dir checkpoints/fconv --tensorboard-logdir tensorboard-logs --log-format simple --log-interval 100 --criterion label_smoothed_cross_entropy  --label-smoothing 0.1
 - Any other relevant information:
